src/panels/config/lovelace/resources/dialog-lovelace-resource-detail.ts

```diff
@@ -101,34 +113,24 @@ export class DialogLovelaceResourceDetail extends LitElement {
             @value-changed=${this._valueChanged}
           ></ha-form>
         </div>
-        ${this._params.resource
-          ? html`
-              <mwc-button
-                slot="secondaryAction"
-                class="warning"
-                @click=${this._deleteResource}
-                .disabled=${this._submitting}
-              >
-                ${this.hass!.localize(
-                  "ui.panel.config.lovelace.resources.detail.delete"
+        <div slot="actions">
+          <mwc-button @click=${this._closeDialog} class="desktop-only">
```

Comment:
It also says:
> Dialog with actions must always have a discard button. On desktop a Cancel button and X-icon, on mobile only the X-icon.

Reply:
I asked @matthiasdebaat for clarification on this 👍

Reply:
Sorry I haven't documented this very well. If we follow the [M3 specs](https://m3.material.io/components/dialogs/guidelines), on mobile we should have a X-icon on the top left and the submit on the top right. A couple of months ago we had some discussions about this, because reaching your finger at the top of the screen is hard.

We decided to have the cancel and submit always visible and at the bottom. To summarise:
- On desktop, there is a X-icon at the top left and cancel/submit buttons at the bottom right.
- On mobile, there is a X-icon at the top left and cancel/submit buttons **sticky** at the bottom right.

I will update our docs accordingly.